ImperatorToCK3/Data_Files/blankMod/output/localization/english/ir_deities_l_english.yml

```diff
@@ -0,0 +1,298 @@
+ ir_lenus:0 "Lenus"
+ ir_intarabus:0 "Intarabus"
+ ir_ancamna:0 "Ancamna"
+ ir_berobreo:0 "Berobreo"
+ ir_glanis:0 "Glanis"
+ ir_eki:0 "Eki"
+ ir_mari:0 "Mari"
+ ir_sucellus:0 "Sucellus"
+ ir_nemetona:0 "Nemetona"
+ # Heptadic / Scythian
+ ir_thagimasadas:0 "Thagimasadas"
+ ir_goitosyrus:0 "Goitosyros"
+ ir_papaios:0 "Papaios"
+ ir_api:0 "Api"
+ ir_tabiti:0 "Tabiti"
+ ir_argimpasa:0 "Argimpasa"
+ ir_targitaos:0 "Targitaos"
+ # Iberic
+ ir_candamius:0 "Candamius"
+ ir_semnocosus:0 "Semnocosus"
+ ir_cariocecus:0 "Cariocecus"
+ ir_ataecina:0 "Ataegina"
+ ir_eacus:0 "Eacus"
+ ir_betatun:0 "Betatun"
+ ir_dercetius:0 "Dercetius"
+ ir_endouellicus:0 "Endovelicus"
+ ir_the_duillae:0 "The Duillae"
+ # Kemetic
+ ir_anhur:0 "Anhur"
+ ir_resheph:0 "Resheph"
+ ir_sekhmet:0 "Sekhmet"
+ ir_horus:0 "Horus"
+ ir_osiris:0 "Osiris"
+ ir_amun:0 "Amun"
+ ir_ptah:0 "Ptah"
+ ir_anubis:0 "Anubis"
+ ir_hathor:0 "Hathor"
+ ir_isis:0 "Isis"
+ ir_ra:0 "Ra"
+ # Khaldic / Armenian
+ ir_shivini:0 "Shivini"
+ ir_theispas:0 "Theispas"
+ ir_khaldi:0 "Ḫaldi"
+ ir_arubani:0 "Arubani"
+ ir_dsvininaue:0 "Dsvininaue"
+ ir_epaninaue:0 "Epaninaue"
+ ir_bagvarti:0 "Bagvarti"
+ ir_huba:0 "Huba"
+ ir_aramazd:0 "Aramazd"
+ ir_vahagn:0 "Vahagn Vishapakagh"
+ ir_nane:0 "Nane"
+ ir_mihr:0 "Mihr"
+ ir_tir:0 "Tir"
+ ir_tsovinar:0 "Tsovinar"
+ ir_anahit:0 "Anahit"
+ ir_spandaramet:0 "Spandaramet"
+ # Megalithic
+ ir_gurzil:0 "Gurzil"
+ ir_sinifere:0 "Sinifere"
+ ir_shaheded:0 "Shaheded"
+ ir_nanna_tala:0 "Nanna Tala"
+ ir_tannit:0 "Tannit"
+ # Zalmoxian
+ ir_pleistoros:0 "Pleistoros"
+ ir_zalmoxis:0 "Zalmoxis"
+ ir_sabazios:0 "Sabazios"
+ ir_zibelthiurdos:0 "Zibelthiurdos"
+ ir_derzelas:0 "Derzelas"
+ ir_bassareus:0 "Bassareus"
+ ir_semele:0 "Semele"
+ ir_bendis:0 "Bendis"
+ ## Possessives
+ # Arabic
+ ir_ailiah_possessive:0 "Allah's"
+ ir_al_kutbay_possessive:0 "Al-Kutba's"
+ ir_al_qaum_possessive:0 "Al-Qaum's"
+ ir_al_uzza_possessive:0 "Al-ʻUzzā's"
+ ir_alilat_possessive:0 "Al-Lāt's"
+ ir_manat_possessive:0 "Manāt's"
+ ir_orotalt_possessive:0 "Orotalt's"
+ ir_ta_lab_possessive:0 "Ta'lab's"
+ # Armazic / Caucasian
+ ir_kopala_possessive:0 "Kopala's"
+ ir_zadeni_possessive:0 "Zaden's"
+ ir_armazi_possessive:0 "Armazi's"
+ ir_ghmerti_possessive:0 "Ghmerti's"
+ ir_dali_possessive:0 "Dali's"
+ ir_ga_possessive:0 "Ga's"
+ ir_adgilis_deda_possessive:0 "Adgilis Deda's"
+ ir_gacim_possessive:0 "Gacim's"
+ ir_ainina_and_danina_possessive:0 "Ainina's & Danina's"
+ # Canaanite
+ ir_anat_possessive:0 "Anath's"
+ ir_el_possessive:0 "ʼĒl's"
+ ir_yam_possessive:0 "Yam's"
+ ir_ba_al_possessive:0 "Baal's"
+ ir_eshmun_possessive:0 "Eshmun's"
+ ir_shapash_possessive:0 "Shapash'"
+ ir_kothar_possessive:0 "Kothar-wa-Khasis'"
+ ir_melqart_possessive:0 "Melqart's"
+ ir_ba_alat_possessive:0 "Baʽalat Gebal's"
+ ir_astarte_possessive:0 "Astarte's"
+ ir_atargatis_possessive:0 "Atargatis'"
+ ir_attar_possessive:0 "Aṯtar's"
+ ir_tanit_possessive:0 "Tanit's"
+ ir_ba_al_hammon_possessive:0 "Baʿal Ḥammon's"
```

Comment:
Encoding error? Does it display properly in the game?

Reply:
CK3 uses entire fonts, so these letters will appear as intended in-game.

Reply:
Interesting, when I reviewed on my phone there were rectangle characters here, but now I'm on PC and it's fine.